kinetic/distribution.yaml

```diff
@@ -5044,6 +5044,14 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2016.4.24-0
     status: maintained
+  libmynteye-release:
```

Comment:
It's a bit strange for the name of the repository to be `libmynteye-release`; this should just be `libmynteye`.